### PR TITLE
solution

### PR DESCRIPTION
**Pull request by abcdesign1**
[DEMO LINK](https://vadym4che.github.io/react_movies-list/)

### CODE REVIEW

**Comment by VadKol** [comment]
image
why are your cards so long?

File: src/components/MovieCard/MovieCard.tsx [hunk 1]
```diff
@@ -1,7 +1,61 @@
...
+            {description}
+          </p>
 
-export const MovieCard: React.FC = () => (
-  <>Put the card here</>
-);
+          <a
+            href={imdbUrl}
+            data-cy="MovieLink"
+          >
```
Comment (VadKol) [inline src/components/MovieCard/MovieCard.tsx:54]: add `target="_blank"`

File: src/components/MoviesList/MoviesList.tsx [hunk 2]
```diff
@@ -1,6 +1,20 @@
...
+import { MovieCard } from '../MovieCard';
 import './MoviesList.scss';
 
-export const MoviesList: React.FC = () => (
-  <>Put the list here</>
+type Props = {
+  movies: Movie[];
+};
+
+export const MoviesList: React.FC<Props> = ({ movies = [] }) => (
```
Comment (VadKol) [inline src/components/MoviesList/MoviesList.tsx:10]:
```export const MoviesList: React.FC<Props> = ({ movies }) => (

**Review by VadKol — CHANGES_REQUESTED** [review]
Fix some bugs

File: src/components/MovieCard/MovieCard.tsx (hunk 1 above)
Comment (vadym4che) [reply src/components/MovieCard/MovieCard.tsx:54]: done

File: src/components/MoviesList/MoviesList.tsx (hunk 2 above)
Comment (vadym4che) [reply src/components/MoviesList/MoviesList.tsx:10]: fixed

File: src/components/MoviesList/MoviesList.tsx [hunk 3]
```diff
@@ -1,6 +1,20 @@
...
+};
+
+export const MoviesList: React.FC<Props> = ({ movies }) => (
+  <div className="movies">
+    {movies.map(movie => (
+      <React.Fragment key={movie.imdbId}>
+        <MovieCard
+          movie={movie}
+        />
+      </React.Fragment>
```
Comment (sergii-nosachenko) [inline src/components/MoviesList/MoviesList.tsx:17]: Useless fragment

```suggestion
      <MovieCard
        movie={movie}
        key={movie.imdbId}
      />
```

File: src/types/Movie.ts [hunk 4]
```diff
@@ -1,5 +1,7 @@
 export interface Movie {
   title: string;
-
-  // add all the other field
+  description: string;
+  imgUrl: string;
+  imdbUrl: string;
+  imdbId?: string;
```
Comment (sergii-nosachenko) [inline src/types/Movie.ts:6]:
```suggestion
  imdbId: string;
```

**Review by sergii-nosachenko — APPROVED** [review]

**Pull request closed without merging**